client/lib/cart-values/cart-items.js

```diff
@@ -404,7 +410,11 @@ function themeItem( themeSlug, source ) {
  * @returns {Object} the new item as `CartItemValue` object
  */
 function domainRegistration( properties ) {
-	return assign( domainItem( properties.productSlug, properties.domain, properties.source ), { is_domain_registration: true } );
+	return assign( domainItem( properties.productSlug, properties.domain, properties.source ),
+		{
+			is_domain_registration: true,
+			...( properties.extra ? { extra: properties.extra } : {} ),
```

Comment:
Then that would suggest that the tests are not actually testing how the system behaves in real life :) But I've promised myself not to touch anything that has to do with the shopping cart, so I'm fine with keeping it like this 😜